sdk/src/Core/Amazon.Runtime/CredentialManagement/AWSCredentialsFactory.cs

```diff
@@ -312,17 +310,18 @@ private static AWSCredentials GetAWSCredentialsInternal(
                                 UserIdentity = options.UserIdentity,
                                 ProfileName = profileName
                             };
-                            return new FederatedAWSCredentials(new SAMLEndpointManager().GetEndpoint(options.EndpointName),
+                            awsCredentials = new FederatedAWSCredentials(new SAMLEndpointManager().GetEndpoint(options.EndpointName),
```

Comment:
Do we need a `FeatureIdSources` here?

Reply:
As of now, the SEP doesn't have a `_PROFILE` feature ID for this credential type, so it's only set in that class constructor: https://github.com/aws/aws-sdk-net/blob/Business-Metrics-Tracking-UA-2.1/sdk/src/Core/Amazon.Runtime/Credentials/FederatedAWSCredentials.cs#L80

(Same comment applies to `SessionAWSCredentials`: https://github.com/aws/aws-sdk-net/blob/Business-Metrics-Tracking-UA-2.1/sdk/src/Core/Amazon.Runtime/Credentials/SessionAWSCredentials.cs#L56)

We can follow up to see if it makes sense to track those as well.